### PR TITLE
[WIP] max command

### PR DESCRIPTION
**Pull request by guilhermeleobas**
Hi @sharkdp 

This still a working in progress and I would like your feedback on what I did. 

Every method implemented in shell-functools takes a `function` as one of its argument but that is not the case with `max/min` methods. Therefore, I had to change a few things to make `max` compatible with the `Command` interface. For instance, one thing I had to change was to make `identity` the default value for the `function` argument in the arguments parser.

Let me know what you think and if everything is Ok, I will create the `min` version as well as some tests. Btw, what do you use for testing? nose? pytest?

Regards

### CODE REVIEW

**Comment by sharkdp** [comment]
@guilhermeleobas Thank you very much for your contribution!

I misunderstood you in #7. I thought you wanted to add a new *function* `max` (instead of a new *command* `max`). If we had a simple function `max :: Int -> Int -> Int`, we could use the `foldl` or `foldl1` commands to compute the maximum:

``` bash
> seq 4 | foldl1 max
4

> echo "1\n3\n7\n4" | foldl1 max
7
```

What do you think?

**Comment by guilhermeleobas** [comment]
That's way simpler than what I did. :)

**Comment by guilhermeleobas** [comment]
Ok, it's done!

File: ft/ft/command.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ def parse_args(self):
 
         self.parse_additional_command_arguments(args)
 
+        # Some functions such as `max` and `min` don't take a function.
```
[the remark this answers is not in the document]
Comment (guilhermeleobas) [reply ft/ft/command.py]: Oh yeah, I forgot to remove this one.

**Pull request merged**